web/app.js

```diff
@@ -2061,6 +2063,22 @@ const PDFViewerApplication = {
     return wholeTicks;
   },
 
+  _accumulateFactor(previousFactor, factor, prop) {
```

Comment:
Given what you're passing to this function, shouldn't the first parameter be named `previousScale` instead?